content/fundamentals/advanced.md

```diff
@@ -0,0 +1,93 @@
+### Advanced Concepts
+
+So far we've breezed over a couple of advanced topics in TypeScript like decorators and how the DI system is actually working under the hood. If all you want is a system that works, ou can probably move on to another section, but if you want to get a deeper understanding of how Nest works, this is the section for you.
+
+#### The @Injectable() Decorator
```

Comment:
```suggestion
#### The Injectable Decorator
```
 if we call it `The Injectable decorator` without the `@` and the parentheses ?

Reply:
to me `@Injectable()` sounds better as the _real decorator_ itself is the return of the function `Injectable()`, which is a decorator factory.

Reply:
Okay 👍